tests/test_register_schema.py

```diff
@@ -53,9 +53,11 @@ def test_reserved_properties():
         el.register_schema({
             '$id': 'test/test',
             'version': 1,
+            'personal-data': False,
```

Comment:
This field isn't in the base schema, correct?

Reply:
Oops, this was supposed to be removed.